src/Layout/Layout.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { Navigation } from '../Navigation/Navigation';
+
+export const Layout: React.FC = () => (
+  <>
+    <Navigation />
+
+    <div className="section">
+      <div className="container">
+        <Outlet />
+      </div>
+    </div>
```

Comment:
Use the container div to wrap some components or pages and avoid using react fragments in some cases

```suggestion
    <div className="section">
        <Outlet />
    </div>
```